### ISSUE TITLE
Will Lemmy communities be able to have ibis wikis within Lemmy?

### ISSUE THREAD

**Issue by slrgt** [Nutomic/ibis#15]
Super excited for ibis (or whatever the official name becomes) and I'm interested in knowing if it will be possible to have wikis within Lemmy so a community can have a section that shows it's wiki without leaving to another website.

Thanks for making this!

**Comment by Nutomic** [comment]
Yes I definitely plan to make it federate with Lemmy, although that will still take a while. Ibis articles could be seen as Lemmy posts, then Lemmy users could read them, comment and possibly even edit the articles. As Lemmy requires posts to be within communities, it could be handled by treating each Ibis instance as a community.

**Comment by slrgt** [comment]
Thats great! Although I meant will it possible to have a wiki within Lemmy itself. So like how reddit communities have their own wiki section, will it be possible to have a wiki on Lemmy

**Comment by K4LCIFER** [comment]
> I meant will it possible to have a wiki within Lemmy itself. So like how reddit communities have their own wiki section, will it be possible to have a wiki on Lemmy

I mean, the community mod could just put a link to the hosted wiki in the about section. It doesn't necessarily have to be contained *within* Lemmy itself.

**Comment by Nutomic** [comment]
This is now implemented, you can see how it works at the following links:
https://lemmy.ml/post/28115476
https://lemmy.ml/c/wiki@test.ibis.wiki
